rsconnect/validation.py

```diff
@@ -0,0 +1,47 @@
+import typing
+
+from rsconnect.exception import RSConnectException
+
+
+def _get_present_options(options: typing.Dict[str, typing.Optional[str]]) -> typing.List[str]:
+    return [k for k, v in options.items() if v]
+
+
+def validate_new_server_options(url, api_key, insecure, cacert, account, token, secret):
+    validate_connection_options(None, url, api_key, insecure, cacert, account, token, secret)
```

Comment:
In general, pass-through methods create extra indirection that we have to reason through, so simplifying them can help a developer save time. In this case, we can make the underlying method use optional keyword arguments, thereby removing the need of a pass-through method.